osf/migrations/0030_preprintprovider_domain.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
I was told to put it in the PR comments and it will go out with the release notes when the release happens